packages/common/src/search/_internal/portalSearchGroups.ts

```diff
@@ -1,24 +1,28 @@
 import { ISearchOptions, searchGroups } from "@esri/arcgis-rest-portal";
 import { IGroup } from "@esri/arcgis-rest-types";
+import { HubError } from "../..";
```

Comment:
I know this was auto added, but import statements should not end in a `.` in order to avoid circular refs.